### DISCUSSION TITLE
Unanswered Ideas

### DISCUSSION THREAD

**Discussion by Illodrus** [ScratchAddons/ScratchAddons#1557] [Ideas] [answered]
Unanswered ideas that could be implemented if someone wants to do them
(post more in comments if more are found)
- #1213 
- #1459
- #987
- #1207 
- #1402
- #1427
- #1518
- #1513

**Comment by TheColaber** [comment, marked as answer]
Please do not make discussions like this one that just dumps all your ideas that you previously made discussions for. I'm sure we have all seen them, including me, but we just did not have a chance to make them yet. After all, addon work is not easy. If you really want something to get implemented, create an issue instead. A discussion is made to discuss a topic. Please however do not make more issues about all of the discussions you have made already. If you really want these to be implemented, try making them yourself. Most are not as easy as you think.

**Reply by Illodrus** [reply]
The goal of the list is to keep old, unused ideas in circulation, not all of which are mine.
It's especially meant for ideas like #1459 that have no comment input and no suggestions from devs.

**Reply by TheColaber** [reply]
That is not our problem. If we do not discuss the discussion, that just means we understand what you are trying to say. Also, posts that say "+1" or "I like this idea" will be marked as offtopic. That is not the point. If you like an idea or want it to be implemented, simply use the thumbs-up emoji. Don't spam emojis. Don't self-emoji yourself. Just please use common sense. These ideas will always be here for devs to view. It may not look that way, but they are. Please understand that this is not the purpose of the discussions. This list is also under https://github.com/ScratchAddons/ScratchAddons/discussions/ FYI. We understand how much you might really want these suggestions implemented, but it not our job to take orders. If I were paid to do what I am doing right now, sure I would make these things. But I'm not paid. No one is. So I have other important things than creating hardcore addons. I have school, homework, and life. That's plenty. Try making an addon yourself, especially one of the ones you said, and understand that addons don't appear in a snap. It takes blood, sweat, and tears packed up with time. Time. If we had more of that you could say everything would be done in a day. Again, please understand. Make issues so that way we can put them in the backlog if we feel like this is a useful wanted feature. Discussions can't be moved to the backlog. Do not make duplicate topics in the discussions tab, nor that issues tab. That's just more of a hassle for us. No comment input means that you've put it in the wrong place. We see it. We go "ok, cool idea". And we live life. I'm not sure what else to say other than the time it took just to explain everything above. And I could go on. Just start paying me like $5 an hour, got it?

**Reply by Illodrus** [reply]
Sorry, I was just trying to make these easier to access and all in one place.

I switched the category to "Ideas" if that helps.

**Reply by pufferfish101007** [reply]
The category doesn't matter - the point is that it takes up unnecessary space and quite frankly we know exactly what we want to be working on, and when we need something new we can just scroll through, or, even better, search for specific things.

**Reply by Illodrus** [reply]
Now you don't have to scroll or search is the point..... if you want this page gone that's fine

**Reply by TheColaber** [reply]
So my question is, what is the point of this? It is not hard to scroll. Like, it's not. Is the point so others can send messages in those discussions? Well, we have all already seen the, and most likely did not respond because we understand your point. Is the point to have spam like this so it becomes harder to see older discussions? Well don't do that, that's just going against your point. Having to scroll down or search is not a big deal. And if this is too keep all important ideas up here, why don't you write them all. They all are important after all. You should then write down every single issue and discussion we have here. Even answered ideas get pilled by other new ones, so this idea of bringing other ideas is not useful to us, and more of spam. Why don't we all make decisions like these promoting some old discussions. I think you see where I am going here.

**Reply by Illodrus** [reply]
You really don't seem to like this........
I looked for a way to delete it but I couldn't find it. You're a member of the repository so if anyone you might be able to, but if not maybe ask @WorldLanguages or someone. Otherwise just downvote I suppose

**Reply by WorldLanguages** [reply]
Don't worry, TheColaber just likes to troll I'd say :P

**Reply by TheColaber** [reply]
> TheColaber just likes to troll I'd say :P

I'd say I'm just trying to deal with future problems. This can stay up, but just no more of these types of discussions please :P

**Comment by rimopa** [comment]
I think #3122 , #3760 , #3770 , #3471 , #3464 , and #2124 are unanswered

**Reply by TheColaber** [reply]
Ideas don't need to be answered... Just questions.
